packages/dev/core/src/Shaders/greasedLine.vertex.fx

```diff
@@ -39,7 +42,8 @@ void main() {
         vec3 grlNext = grl_nextAndCounters.xyz;
         grlCounters = grl_nextAndCounters.w;
 
-        mat4 grlMatrix = worldViewProjection;
+        mat4 grlMatrix = worldViewProjection * finalWorld;

```

Comment:
It should be `viewProjection * finalWorld` instead.

Reply:
thank you, fixed

Reply:
sry, one more thing to fix

Reply:
@Popov72 
Is this correct? Can I drop these three uniorms?
```javascript
                uniforms: [
                    // "worldViewProjection",
                    // "worldView",
                    // "view",
                    "world",
                    "viewProjection",
                    "projection",
```

Reply:
Yes, if you don't use them yourself you can remove them.

Reply:
OIk, thank you!